### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'replace'

### ISSUE THREAD

**Issue by Amadeus-** [bbolli/tumblr-utils#188]
I'm seeing this exception several times when running tumblr_backup.py (commit:  55b8301):

```
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "tumblr_backup.py", line 1078, in handler
    work()
  File "tumblr_backup.py", line 691, in save_content
    append_try('body')
  File "tumblr_backup.py", line 677, in append_try
    self.get_inline_video_poster, elt
  File "/usr/lib/python2.7/re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "tumblr_backup.py", line 877, in get_inline_video_poster
    saved_name, match.group(3)
AttributeError: 'tuple' object has no attribute 'replace'
```

**Comment by Amadeus-** [comment]
Also, I tried reverting back to commit b93eab9 and it runs better (i.e., it doesn't hang completely like 55b8301 does; however, it does have this recurring error on some blogs:

```
Exception in thread Thread-13:                  
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "tumblr_backup.py", line 1043, in handler
    work()
  File "tumblr_backup.py", line 781, in save_content
    self.save_post()
  File "tumblr_backup.py", line 955, in save_post
    f.write(self.get_post())
  File "tumblr_backup.py", line 910, in get_post
    typ = ('liked-' if options.likes else '') + self.typ
TypeError: cannot concatenate 'str' and 'NoneType' objects
```

**Comment by Cebtenzzre** [comment]
It looks like aggroskater already discovered and [fixed](https://github.com/aggroskater/tumblr-utils/commit/b09e2554c519554bc9975fe695b8037566035025) this issue in his fork.  I'd prefer parentheses to a temporary variable though.

**Comment by Amadeus-** [comment]
I should have mentioned this, but the exception mentioned in my original posting on this issue (the bug in  55b8301) causes the process to hang completely.   So, that's why I reverted back and saw that older exception, which I guess is now fixed in the latest version.

**Comment by aggroskater** [comment]
Should be fixed with 85b4ce3dafd2ff2916f713b41e7d22a363990c12.
